tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -153,5 +153,178 @@ public void run() {
   public Boolean shouldRun() {
     return !lock.isLocked();
   }
-}
 
+  public ReviewerConfig printStartupOsReviewerConfig() throws IOException {
+    String localRegistryFilePath = fileUtils.joinPaths(fileUtils.getCurrentWorkingDirectory(),
+        "reviewer_config.prototxt");
+    ReviewerConfig reviewerConfig = (ReviewerConfig) fileUtils.readPrototxt(localRegistryFilePath,
+        ReviewerConfig.newBuilder());
+    //System.out.println("StartupOs ReviewerConfig:\n" + reviewerConfig.toString());
+    return reviewerConfig;
+  }
+
+  public ReviewerConfig printHasadnaReviewerConfig() throws IOException {
+    String targetDirectory = fileUtils.expandHomeDirectory("~/hasadna");
+    String localHasadnaRegistryFilePath = fileUtils.joinPaths(targetDirectory, "reviewer_config.prototxt");
+    ReviewerConfig reviewerConfigHasadna = (ReviewerConfig) fileUtils.readPrototxt(localHasadnaRegistryFilePath,
+        ReviewerConfig.newBuilder());
+    //System.out.println("Hasadna ReviewerConfig:\n" + reviewerConfigHasadna.toString());
+    return reviewerConfigHasadna;
+  }
+
+  public void compareReviewerConfigData(ReviewerConfig reviewerConfig1, ReviewerConfig reviewerConfig2) throws IOException {
+    String displayName = reviewerConfig1.getDisplayName();
+    LinkedHashSet<Repo> repoList = new LinkedHashSet<Repo>();
```

Comment:
Explicit type argument Repo can be replaced with <>
`new LinkedHashSet<Repo>()` -> `new LinkedHashSet<>()`

Also below.

Reply:
Is that true for all the types I used in `LinkedHashSet`s?